test/compilable/testdefines.d

```diff
@@ -1,4 +1,14 @@
 // EXTRA_FILES: imports/defines.c
 import imports.defines;
 
-static assert(GHI == 3);
+static assert(CC == 'c');
+static assert(I32 == 3);
```

Comment:
Can you test if typing precist, or at least do a single test for typing and on the others, don't use implicit conversion?

Reply:
The asserts don't rely on implicit conversion.

Reply:
> The asserts don't rely on implicit conversion.

e.g.:
```d
static assert(U32 == 4);
```
to
```d
static assert(U32 == 4U);
```

What I mean is, if we get to a point where `U32` is silently converted, some weird behaviour may happen. This is very unlikely tho, but I think we lose nothing testing those types.